### ISSUE TITLE
Breadcrumbs keyboard navigation does not work

### ISSUE THREAD

**Issue by fejx** [microsoft/vscode#98321]
- VSCode Version: 1.46.0-insider
- OS Version: Windows_NT x64 10.0.19041

According to [the documentation](https://code.visualstudio.com/Docs/editor/editingevolved), it should be possible to navigate through the breadcrumbs with the left and right arrow keys:

> ### Breadcrumb keyboard navigation
> 
> To interact with breadcrumbs, use the Focus Breadcrumbs command or press `Ctrl+Shift+.`. It will select that last element and open a dropdown that allows you to navigate to a sibling file or symbol. Use the `Left` and `Right` keyboard shortcuts to go to elements before or after the current element. When the dropdown appears, start typing - all matching elements will be highlighted and the best match will be selected for quick navigation.
> You can also interact with breadcrumbs without the dropdown. Press `Ctrl+Shift+;` to focus the last element, use `Left` and `Right` to navigate, and use `Space` to reveal the element in the editor.

If I focus the breadcrumbs using `Ctrl+Shift+;`, the arrow keys left and right work, but as soon as I select one of the elements, they do not work anymore. If I focus the breadcrumbs using `Ctrl+Shift+.`, the current element gets selected right away, so the left/right navigation fails again.

*Follow-up on #98315.*

Steps to Reproduce:

1. Open a new file `test.js` with the following content:
```js
function a() {}
```
2. Execute the command "Focus Breadcrumbs" (`Ctrl+Shift+.` by default).
3. Using right and left arrow keys does not do anything.

Does this issue occur when all extensions are disabled?: **Yes**

**Comment by jrieken** [comment]
Yeah, that a problem with the tree eating up left/right. You can workaround like this

* use `ctrl+space+;` to only focus the breadcrumbs without showing the breadcrumbs picker
* use `ctrl+left` or `ctrl+right` to move inside breadcrumbs while the picker is showing
